Documentation/migration-issues.md

```diff
@@ -1,3 +1,21 @@
+## Migration from project.json/xproj to csproj
+
+* For Windows:
+   * Install Visual Studio 2017 RC with the .NET Core workload
+   * Launch Visual Studio 2017 RC
+   * Open your sln or project containing global.json or project.json/xproj
+   * Build 
+   * Run
+
+* For Non-Windows:
+   * Install .NET Core 1.1 SDK
```

Comment:
I am not sure this is correct. What is this 1.1 SDK. I believe our current MSBuild 1.0.3 + 1.1 runtime sdk ships with the name of 1.0.3 SDK. Also, I would argue that providing links here would be usedful.